packages/office-ui-fabric-react/src/components/ColorPicker/ColorPicker.base.tsx

```diff
@@ -118,8 +119,11 @@ export class ColorPickerBase extends React.Component<IColorPickerProps, IColorPi
     const props = this.props;
     const strings = this._strings;
     const textLabels = this._textLabels;
-    const { theme, className, styles, alphaSliderHidden } = props;
+    const { theme, className, styles, alphaSliderHidden, showAlphaAsTransparencySlider } = props;
     const { color } = this.state;
+    const alphaValue = color.a;
+    const sliderValue = alphaValue ? this._getSliderValue(alphaValue) : 100;
+    textLabels.a = showAlphaAsTransparencySlider ? props.transparencyLabel || strings.transparency : props.alphaLabel || strings.alpha;
```

Comment:
Please don't mutate `textLabels`! (And in general, you shouldn't be doing side effects like this in the render method.)

Instead, update the type of `textLabels` like this, and set `transparency` along with the other values in the constructor. 
`private _textLabels: { [K in keyof IRGBHex]: string; transparency: string; };`

Then update the logic here to decide which of `textLabels` to use.